csm/core/blogic/const.py

```diff
@@ -490,8 +490,12 @@
 AVAILABLE = 'avail'
 USAGE_PERCENTAGE = 'usage_percentage'
 
+# CSM conf entry
+CSM='CSM'
+
 # Keys for  Description
 DECRYPTION_KEYS = {
+    "CSM.password": "csm",
```

Comment:
NON_ROOT_USER_PASS is still there const.py.

Reply:
yes will remove it.